RecoTauTag/Configuration/python/RecoTauTag_EventContent_cff.py

```diff
@@ -112,7 +113,9 @@
         'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationNeutralIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationPUcorrPtSum_*_*',
-        'keep *_hpsPFTauTransverseImpactParameters_*_*'
+        'keep *_hpsPFTauTransverseImpactParameters_*_*',
+        'keep *_hpsPFTauNeutralIsoPtSumWeight_*_*',
+        'keep *_hpsPFTauPhotonPtSumOutsideSignalCone_*_*'
```

Comment:
@slava77 Sorry about that - I neglected to remove them. I'll do it now. Just to be sure - is it just lines 117-118, or is it 116-118?